hy/__init__.py

```diff
@@ -19,7 +19,11 @@
 # DEALINGS IN THE SOFTWARE.
 
 
-from hy.version import __version__, __appname__  # NOQA
+__appname__ = 'hy'
+try:
+    from hy.version import __version__
+except ImportError:
+    __version__ = 'unknown'
```

Comment:
Wouldn't that re-create the possibility of somebody thinking (or leading us to think) that they're on 0.11.0 (or whatever) when they're not?